addons/scroll-project-titles/style.css

```diff
@@ -0,0 +1,4 @@
+.project-title.no-edit {
+  overflow-x: scroll !important;
+  text-overflow: clip !important;
```

Comment:
I don't think `!important` necessary on thse two lines.

Reply:
Thanks, fixed it.